ui/src/components/harvest-games/HarvestGameInstructions.js

```diff
@@ -0,0 +1,357 @@
+import {
+  Box,
+  Container,
+  Select,
+  Heading,
+  Input,
+  Button,
+  VStack,
+  Image,
+  HStack,
+  Grid,
+} from '@chakra-ui/react';
+import { Text } from '@chakra-ui/layout';
+import { useState, Fragment } from 'react';
+
+import './styles.css';
+
+export default function HarvestGameInstructions() {
+  const gameType = [
+    {
+      game_num: 'Game #1',
+      name: 'SMUGGLING THROUGH SECURITY',
+      round: 3,
+      player: 5,
+      points: 45,
+      materials: '-',
+      imageUrl: [
+        'https://hongkong.sub.hmcc.net/wp-content/uploads/hg1_1.gif',
+        'https://hongkong.sub.hmcc.net/wp-content/uploads/hg1_2.gif',
+      ],
+      id: 'game_1',
+      pwd: 'Smuggling',
+    },
+
+    {
+      game_num: 'Game #2',
+      name: 'THE PACKING GAME',
+      round: 2,
+      player: 2,
+      points: 230,
+      materials: [
+        '10 Shirts',
+        '6  SweatPants',
+        '6  Books',
+        '6  Hoodies',
+        '9  Socks',
+        '6  Shoes',
+        '8  Hats',
+        '6  Water bottles',
+      ],
+      imageUrl: [
+        'https://hongkong.sub.hmcc.net/wp-content/uploads/hg2_1.gif',
+        'https://hongkong.sub.hmcc.net/wp-content/uploads/hg2_2.gif',
+        'https://hongkong.sub.hmcc.net/wp-content/uploads/hg2_3.gif',
+        'https://hongkong.sub.hmcc.net/wp-content/uploads/hg2_4.gif',
+      ],
+      id: 'game_2',
+      pwd: 'Packing',
+    },
+
+    {
+      game_num: 'Game #3',
+      name: 'THE AIRPLANE RELAY RACE',
+      round: 4,
+      player: 5,
+      materials: '-',
+      points: '-',
+      imageUrl: [
+        'https://hongkong.sub.hmcc.net/wp-content/uploads/hg3_1.gif',
+        'https://hongkong.sub.hmcc.net/wp-content/uploads/hg3_2.gif',
+        'https://hongkong.sub.hmcc.net/wp-content/uploads/hg3_3.gif',
+      ],
+      id: 'game_3',
+      pwd: 'Airplane',
+    },
+  ];
+
+  const [selected, setSelected] = useState(0);
+  const [password, setPassword] = useState('');
+
+  const handleSubmit = () => {
```

Comment:
no error handling if password is wrong? i guess that works too but maybe some form of feedback to tell users that they have entered smth wrong? maybe `alert()`